docker-compose.yaml

```diff
@@ -115,6 +115,8 @@ services:
     volumes:
       - grafana_data:/var/lib/grafana
     env_file: .env
+    environment:
+      - GF_SERVER_ROOT_URL=http://${GRAFANA_DOMAIN:-149.7.4.141}:${GRAFANA_PORT:-30001}/
```

Comment:
Why is `149.7.4.141` hardcoded ?

Reply:
It just defaults to this, but we can of course pass in the our own IP via the `GRAFANA_DOMAIN` env var

If we don't want this codeified at all I can remove it.

Reply:
Yeah, I would suggest to not have any explicit hardcoded IP on it